src/components/movie-card/movie-card.jsx

```diff
@@ -8,4 +11,12 @@ export const MovieCard = ({ movie, onMovieClick }) => {
       {movie.title}
     </div>
   );
-};
\ No newline at end of file
+};
+
+// Define all the props constraints for the MovieCard
+MovieCard.propTypes = {
+  movie: PropTypes.shape({
+    title: PropTypes.string
```

Comment:
The attribute name is incorrect. It's `Title` according to what was defined in the API call.